.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,111 @@
+name: Bug Report
+description: Released feature does not work as expected in the SAP Cloud SDK for AI (Java).
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This repository is for Java related issues only. Please open [JavaScript / TypeScript issues here](https://github.com/SAP/ai-sdk-js/issues/new/choose)
+        For any issue with the AI Core client, use the [Slack Support Channel for AI Core](https://sap-ml.slack.com/archives/C01DZC6EGB0) then simply type `/support-request`
+        **Missing information may result in delayed response**
```

(Minor)

I don't think we can link internal support channels from public resources.

```suggestion
        This repository is for Java related issues only. Please open [JavaScript / TypeScript issues here](https://github.com/SAP/ai-sdk-js/issues/new/choose)
        **Missing information may result in delayed response**
```

---

It stays, this is the only sentence I wanted to put in the template